index.html

```diff
@@ -472,7 +472,13 @@ <h3>StatusList2021Credential</h3>
 The <code>encodedList</code> property of the credential <a>subject</a> MUST be
 the GZIP-compressed [[RFC1952]], base-64 encoded [[RFC4648]] bitstring values
 for the associated range of <a>verifiable credential</a> status values. The
-uncompressed bitstring MUST be at least 16KB in size.
+uncompressed bitstring MUST be at least 16KB in size. The bitstring MUST be
+encoded as a big-endian value with a start index value of zero
+(<code>0</code>) located at the left-most bit in the bitstring and the
+end index of one less than the length of the bitstring
+(<code>bitstring_length - 1</code>) and located at the right-most bit in the
+bitstring. Further information on bitstring encoding can be found in Section
+<a href="#bitstring-encoding"></a>.
```

Comment:
```suggestion
uncompressed bitstring MUST be at least 16KB in size. The bitstring MUST be
encoded such that the first index, with a value of zero
(<code>0</code>), is located at the left-most bit in the bitstring and the
last index, with a value of one less than the length of the bitstring
(<code>bitstring_length - 1</code>), is located at the right-most bit in the
bitstring. Further information on bitstring encoding can be found in Section
<a href="#bitstring-encoding"></a>.
```

Reply:
We understand `bitstring_length - 1` as written.  Later readers will benefit by a contrived example that shows, for instance, `bitstring_length = 17000` and `last a/k/a right-most index = 16999`